### ISSUE TITLE
Rearrange the dropdowns of the Service task implementation 

### ISSUE THREAD

**Issue by ingorichtsmeier** [bpmn-io/bpmn-js-properties-panel#597]
__What should we do?__

Order the options of implementing a _service task like_ regarding to our new best practices: Remote architecture first, avoid Connectors if possible: https://docs.camunda.io/docs/components/best-practices/development/invoking-services-from-the-process-c7/

Sequence should be:

1. (DMN)
1. External
1. Java class
1. Expression
1. Delegate Expression
1. Connectors

**Never** sort them by alphabet: https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/src/provider/camunda-platform/properties/ImplementationTypeProps.js#L206

__Why should we do it?__

Reduce developer confusion by providing a well sorted selection for implementation. As a developer I want to find the most important values on the head of the list.

**Comment by andreasgeier** [comment]
@ingorichtsmeier 

I trust your judgment and would follow your suggestion of the changed order. But I wonder why you are requesting the change only for the `service task`. What about `send task` and `business rule task`? (Are there other symbols with type select?) Currently for these tasks we display the same list of types in a drop down menu. Does the change make sense for them, too?

**Comment by nikku** [comment]
It does make sense for all _service task like_ elements, i.e. also for intermediate message throw events (but also send task and business rule task).

**Comment by MaxTru** [comment]
Moving to ready since relevant for April Release (and very easy fix as well)

**Comment by ingorichtsmeier** [comment]
Hi @andreasgeier, you are right. 

I found the issue when I built an example and I was just using a Service task. Then I inspected the code and found that a change in these lines (https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/src/provider/camunda-platform/properties/ImplementationTypeProps.js#L184-L207) may be sufficient to get the change done.

I assumed that the code is reused for all task and event types that require an implementation: 
* service task
* send task
* business rule task
* intermediate message throw event
* message throw end event

These elements should be tested afterwards.
